examples/constraint-validations/constraint_validations.md

```diff
@@ -0,0 +1,11 @@
+# Hello world main
```

Comment:
Lets change this

Reply:
Fixed vis [732d286](https://github.com/ballerina-platform/ballerina-distribution/pull/3060/commits/732d2860a98863b093c74da6f1ec67631c29e11d)